themes/dark/yaru-dark.json

```diff
@@ -356,9 +356,9 @@
     "list.inactiveSelectionBackground": "#575757",
     "activityBar.background": "#575757",
```

Comment:
I'm thinking that `activityBar.background` and `input.background` should be darkened to `#484848` to improve contrast against the new badge color. I got `#484848` from the Software Center.

![badge-yaru-dark-contrast](https://user-images.githubusercontent.com/4039042/87612668-28076e80-c6d9-11ea-86b0-3e36276e5200.png)